edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -787,6 +794,38 @@ def test_access_granted_past_due_exam_no_attempt(self):
         self.assertEqual(200, response.status_code)
         self.assert_valid_exam_access_token(response, self.user, self.past_due_exam)
 
+    @ddt.data(
+        (timedelta(minutes=20), timedelta(minutes=0), 200, 60),  # exam attempt time remaining more than default exp
+        # todo: calculate and change this to be testing less than default exp
```

Comment:
Todo: I need to calculate and reset the values here to test that the expiration window is correct when the expiration window is set to be different than the default (when the time remaining is less than the default exp window). That being said, let me know if this test approach makes sense! Thanks.